components/profile-picture/ProfilePicture.tsx

```diff
@@ -0,0 +1,102 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useEffect, useState } from "react";
+import { StyleSheet, View, Image, Pressable, Animated } from "react-native";
+import { LinearGradient } from "expo-linear-gradient";
+import * as Animatable from "react-native-animatable";
+
+type ProfilePictureProps = {
+  imageSize: number;
+  borderColor: string[];
+  openStories: () => void;
```

Comment:
Do all profile pictures open stories?
I would say this is a specific prop name for a more generic component.
Receiving `onPress` prop seems more appropriate - whoever renders it decides what happens when it's pressed